### DISCUSSION TITLE
Suggestion: topics about various horror films & franchises (For Spoopy Month)

### DISCUSSION THREAD

**Discussion by JohnSmith2007** [Monika-After-Story/MonikaModDev#7993] [general]
Stuff like Nightmare on Elm Street, Halloween, Final Destionation, etc. (Personally, i think it'd be funny to hear Monika talking about how some of the daths in the Final Destination movies are more funny than scary)  

**cough** Tanning Salon scene from Final Destination 3 **cough**

**Comment by Booplicate** [comment]
For future, for **mod** suggestions and bug reports open issues.
